src/components/heading/__spec__.js

```diff
@@ -74,7 +74,7 @@ describe("Heading", () => {
     );
 
     const link = wrapper.find(Link);
-    link.find("button").simulate("mousedown");
+    link.find("a").simulate("mousedown");
```

Comment:
This is interesting, so the back arrow will now render as an `a`. We should check if we should be using a button here or an anchor.

![image](https://user-images.githubusercontent.com/2328042/105724779-21777080-5f20-11eb-94ab-3a5f289ed6d4.png)

Reply:
As discussed elsewhere we should be using an `a` here